AUTHORS

```diff
@@ -2253,6 +2261,7 @@ VladimirAus <v_roudakov@yahoo.com>
 Vladislav Kolesnikov <vkolesnikov@beget.ru>
 Vlastimil Zeman <vlastimil.zeman@diffblue.com>
 Vojtech Vitek (V-Teq) <vvitek@redhat.com>
+voloder <110066198+voloder@users.noreply.github.com>
```

Comment:
😢 Looks like this one went unnoticed (handle, not name)

Reply:
This is intentional; the user does not have 'better' data on their GitHub profile.

Reply:
Yeah, we should've ask them to provide when merging the PR.

Reply:
I don't see why -- if a user wants to be fully pseudonymous, that's their right. Plenty of other examples of that in this file, too.

Reply:
https://github.com/moby/moby/blob/master/CONTRIBUTING.md#sign-your-work

> Use your real name (sorry, no pseudonyms or anonymous contributions.)

Reply:
Ah, well, fair enough... That ship has sailed, I guess.